sections/node.zsh

```diff
@@ -9,6 +9,7 @@
 # ------------------------------------------------------------------------------
 
 SPACESHIP_NODE_SHOW="${SPACESHIP_NODE_SHOW=true}"
+SPACESHIP_NODE_SHOW_IN_HOME="${SPACESHIP_NODE_SHOW_IN_HOME=true}"
```

Comment:
Looking at the discussion in the referenced bug I wonder if this option should be `false` by default, or should it even exist - can you think of an example where someone would _want_ to see the version in $HOME dir?

Reply:
You're right, I can't think of any such reason. What should we do? Hide it on `$HOME` always or give an option and set it to `false` by default?

Reply:
Give an option and set it to `false` by default.

Reply:
There is such thing as having too many options 😛

Reply:
Welp. Now I'm conflicted here.  :open_mouth:

Reply:
I don't care much, make @salmanulfarzy happy 😉

Reply:
On it.

Reply:
Currently _most_ of the things in `spaceship` is configurable. IMO it's better to have it now or wait until some user opens an issue asking the option :man_shrugging: